### PR TITLE
[DOCS] Private LLMs & free-tier is not likely enough

### PR DESCRIPTION
**Pull request by lucabelluccini**
- Add a statement about not supporting Local LLMs
- Add a statement regarding free-tier LLMs are likely not enough to power AI Assistant

### CODE REVIEW

File: docs/en/observability/observability-ai-assistant.asciidoc [hunk 1]
```diff
@@ -41,6 +41,14 @@ The AI assistant requires the following:
 ** AWS Bedrock, specifically the Anthropic Claude models.
 * The knowledge base requires a 4 GB {ml} node.
 
+[IMPORTANT]
+====
+The free tier offered by third-party generative AI provider may not be sufficient for the proper functioning of the AI assistant.
+In most cases, a paid subscription to one of the supported providers is required.
+
+The Observability AI assistant doesn't support or recommend connecting to private a LLM.
```
Comment (alaudazzi) [inline docs/en/observability/observability-ai-assistant.asciidoc:49]:
```suggestion
The Observability AI assistant doesn't support or recommend connecting to a private LLM.
```

**Review by alaudazzi — DISMISSED** [review]
Left a minor editing suggestion, otherwise LGTM.

**Comment by lucabelluccini** [comment]
Thank you @alaudazzi - I actually realized my copy paste was incomplete. I've submitted another change.

**Review by bmorelli25 — APPROVED** [review]
Thanks, Luca!

**Comment by lucabelluccini** [comment]
Hello @bmorelli25 - is it ok to merge the backports?

**Pull request merged**